network/transport/grpc/connection_list.go

```diff
@@ -0,0 +1,73 @@
+package grpc
+
+import (
+	"github.com/nuts-foundation/nuts-node/network/transport"
+	"sync"
+)
+
+type connectionList struct {
```

Comment:
I missed this one when refactoring inbound connections, because outbound is not a problem (since they're orchestrated by configuring bootstrap nodes). Will fix.